app/models/group.rb

```diff
@@ -145,6 +145,10 @@ def self.visibility_levels
     @visibility_levels = Enum.new(public: 0, logged_on_users: 1, members: 2, staff: 3, owners: 4)
   end
 
+  def self.auto_group_range(lower, upper)
+    (Group::AUTO_GROUPS[lower.to_sym]..Group::AUTO_GROUPS[upper.to_sym]).to_a
+  end
```

Comment:
Feel slightly iffy that we name it `_range` but it returns an array instead of a range. 🙂 

Also there's a small change of `RangeError` here if one of the bound keys isn't in `AUTO_GROUPS` since an endless range can't be turned into an array.

Maybe we can do a check for keys, or we can rescue `RangeError` and return an empty array.

Reply:
Haha I thought someone would call me out on that _range thing :P but yes will slightly fix this